arch/arm/src/stm32f7/stm32_qspi.c

```diff
@@ -2066,16 +2066,19 @@ static int qspi_command(struct qspi_dev_s *dev,
           ret = qspi_receive_blocking(priv, &xctn);
         }
 
+      /* Wait for Transfer complete*/
+
+      qspi_waitstatusflags(priv, QSPI_SR_TCF,1);
```

Comment:
I think this should be leaved to driver to wait for transfer done this time, or next time before any other operations, to get better performance. There is BUSY flag in status register. Is this the same wait as that BUSY flag?

Reply:
This flag is related to data transmission,  and BUSY is related to more operations , instructions or addresses for example, the TCF need be cleared by set CTCF bit in FCR, and BUSY bit  clears automatically when the operation with the Flash memory is finished and the FIFO is empty.

The wait for transfer complete operation is contained in this qspi_command() function, i move it after data transfer, otherwise, it will in dead loop to wait transfer complete if only a command be transmitted, like spi flash reset (0xFF) for example.

Reply:
So could the loop in driver to check status register value be removed?

Reply:
I've verified and it works well. Please do more test to make sure we can remove the loop workarond in w25_write_status_locked().